Resources/apiresources.robot

```diff
@@ -855,3 +855,51 @@ fetchJsonRespContainingAHUPropertiesOfSpecificGroup
     ${query}=    gqlQueries.getAHUsPropertiesOfSpecificGroup  ${group_name}
     ${json_dictionary}=  gqlFetchJsonResponseFromQuery     ${query}
     return from keyword    ${json_dictionary}
+
+#=======added for Alarm testcases
+    #Gets the list of message for specified alram tpye
+getAllAlarmMessagesOfSpecifiedAlarmType
+    [Arguments]  ${alarm_type}
+    @{ahu_list}=    apiresources.getAHUNamesListOfGroup
+    log to console  ==================List of Ahu in the grpup is ${ahu_list}
+    ${ahu_count}=      apiresources.getAHUCount
+    log to console  ===================Count of Ahu in the Group ${ahu_count}
+    #fetching the Alarm  message for Ahu and putting in list
+    @{alarm_message_list}=    Create List
+    FOR    ${i}    IN RANGE   ${ahu_count}
+        ${query}=    gqlQueries.getAlarmStatusQuery     ${ahu_list}[${i}]   ${alarm_type}
+        ${json_dictionary}=  gqlFetchJsonResponseFromQuery     ${query}
+        Append To List    ${alarm_message_list}    ${json_dictionary['data']['alarms'][0]['message']}
+    END
+    log to console      ==================List of message each ahu for the alarm ${alarm_message_list}
+    return from keyword    ${alarm_message_list}
+
+    #Gets the list of the Ahu-pathname(ex:NoBindings / NB-AHU-10 / SyncFaultStatus) that are in mistmatch
+getAllAHUInGroupInMismatchState
+    ${query}=    gqlQueries.getAHUsInMismatchState
+    ${json_dictionary}=  gqlFetchJsonResponseFromQuery     ${query}
+    log to console  ========full response ${json_dictionary}
+    #fetching the pathname that has groupname, Ahuname and putting in list
+    ${total}=    get length   ${json_dictionary['data']['site']['groups'][0]['nameahus']}
```

Comment:
Total length should be fetched for AHUs that has $.data.site.groups[0].nameahus[0].pointCurrent[0].SyncFaultStatus.value as 1 or 2.
if value is 0,then they are not in mismatch state